src/style.css

```
@@ -0,0 +1,106 @@
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.main__block {
+  display: flex;
+  flex-basis: 25.1%;
```

```suggestion
  flex-basis: 33.33%;
```
Let's make it more logical. You should split your container to three parts, so 33% would be more clear and meaningful